radio/src/targets/taranis/lcd_driver_spi.cpp

```diff
@@ -205,6 +225,9 @@ void lcdRefresh(bool wait)
     lcdWriteCommand(0x04);
 #endif
 
+#if defined(PCBX7ACCESS)
```

Comment:
I would prefer we test if we have DMA on LCD, like this

`#if defined(LCD_DMA)`

Reply:
LCD has DMA but doesn't work : (